### DISCUSSION TITLE
Lazy loading the components are throwing errors. Failing to load components in vue/nuxt-storybook

### DISCUSSION THREAD

**Discussion by saikiran-dev** [storybookjs/storybook#13569] [Help]
I'm trying lazy loading the child components in the parent component which storybook is loading as a story and I'm getting this error in the storybook console
 
 But it is absolutely fine in my local dev.
 
![story_index](https://user-images.githubusercontent.com/54679219/103757966-8a617c00-5037-11eb-986a-636e580c31cd.PNG)

**Comment by shilman** [comment]
which version of webpack?
